tsfpga/module.py

```diff
@@ -159,7 +166,7 @@ def registers(self) -> Optional[RegisterList]:
             # Only create object from TOML once.
             return self._registers
 
-        toml_file = self.path / f"regs_{self.name}.toml"
+        toml_file = self.register_data_file
```

Comment:
It worked out fine to override this property with our own implementation.

In our case we have actually wrapped the register generation in an object which is a member of our own defined class.